src/framed/std/io.clj

```diff
@@ -96,3 +96,13 @@
   (if (.exists (io/file dest))
     (throw (IllegalArgumentException. "dest already exists"))
     (move' src dest)))
+
+(defn copy
+  "Copy file-like src to file-like dest and return dest.
+   Throws error if dest exists and is non-empty"
```

Comment:
"exists and non-empty" seems redundant; it seems like "non-empty" implies exists?